mmdet3d/models/decode_heads/minkunet_head.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List
+
+import torch
+from torch import Tensor
+from torch import nn as nn
+
+from mmdet3d.models.layers import IS_TORCHSPARSE_AVAILABLE
+from mmdet3d.registry import MODELS
+from mmdet3d.structures.det3d_data_sample import SampleList
+from .decode_head import Base3DDecodeHead
+
+if IS_TORCHSPARSE_AVAILABLE:
+    from torchsparse import SparseTensor
+else:
+    SparseTensor = None
+
+
+@MODELS.register_module()
+class MinkUNetHead(Base3DDecodeHead):
+    r"""MinkUNet decoder head with TorchSparse backend.
+
+    Refer to `implementation code <https://github.com/mit-han-lab/spvnas>`_.
+
+    Args:
+        channels (int): The input channel of conv_seg.
+        num_classes (int): Number of classes.
+    """
+
+    def __init__(self, channels: int, num_classes: int, **kwargs) -> None:
+        super().__init__(channels, num_classes, **kwargs)
+        self.conv_seg = nn.Linear(channels, num_classes)
```

Comment:
In the latest code, we may implement `build_conv_seg` instead of using `self.conv_seg=nn.Linear(channels, num_classes)`